modules/friendlytag.js

```diff
@@ -1457,6 +1498,11 @@ Twinkle.tag.callback.evaluate = function friendlytagCallbackEvaluate(e) {
 		return;
 	}
 
+	if(params.tags.indexOf('expand language') !== -1 && params.tagParameters.expandLanguageLangCode == null) {
```

Comment:
Doesn't work, see code for `cleanup`:

```suggestion
	if( params.tags.indexOf('expand language') !== -1 && params.tagParameters. expandLanguageLangCode.trim && params.tagParameters. expandLanguageLangCode.trim() === "") {
```

Reply:
fixed I think. The check for `params.tagParameters.expandLanguageLangCode.trim` (function type) isn't needed per #486. All browsers now support trim() function.